android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -504,6 +504,20 @@ private PackageInfo getPackageInfo() throws Exception {
     return getReactApplicationContext().getPackageManager().getPackageInfo(getReactApplicationContext().getPackageName(), 0);
   }
 
+  private String getInstallerPackageNameSync() {
+    String packageName = getReactApplicationContext().getPackageName();
+    String installerPackageName = getReactApplicationContext().getPackageManager().getInstallerPackageName(packageName);
+
+    if (installerPackageName === null) {
```

Comment:
```suggestion
    if (installerPackageName == null) {
```

triple-equals is a javascript-ism, I don't think it will compile in Java?

Reply:
You're right, i'm in JS land :D